src/lib/y2network/routing_table.rb

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) [2019] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+module Y2Network
+  class RoutingTable
+    attr_reader :routes
+
+    # MAIN_TABLE_ID = 254
```

Comment:
???

Reply:
You can have several routing tables in a system an the main one has 254 as ID. I will remove the comment as it is not useful right now.

Reply:
It could be useful if we add support for policy routing and `ip rules`, but that is not supported in YaST so far. So by now we should probably let it out of the scope.